hindsight/modules/heka/msg_interpolate.lua

```diff
@@ -0,0 +1,99 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
not needed this is also in the sandbox
http://mozilla-services.github.io/lua_sandbox/modules/heka/msg_interpolate.html